sdk/core/core-tracing/src/createSpan.ts

```diff
@@ -32,6 +33,20 @@ export interface CreateSpanFunctionArgs {
   namespace: string;
 }
 
+function isTracingDisabled(): boolean {
+  if (typeof process === "undefined") {
+    // not supported in browser for now without polyfills
+    return false;
+  }
+
+  if (process.env.AZURE_TRACING_DISABLED === "false") {
```

Comment:
Torn on this. On the one hand, what about "0", "off", etc - do we want to special-case all of them? On the other hand, seems like "false" might be pretty common. I Can go either way - let me know if you have strong opinions!

Reply:
This is what .NET is doing at least: https://github.com/Azure/azure-sdk-for-net/blob/a0bfd542d9b0f74059d6ad3682a06c926e7ae760/sdk/core/Azure.Core/src/DiagnosticsOptions.cs#L111

Their supported values seems reasonable to me. I'm fine with us defaulting to Boolean() after checking for those special values

Reply:
Yea, I think I'm happy with consistency - I'll make sure that we cover the same values so its consistent across languages